src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java

```diff
@@ -274,15 +276,16 @@ public boolean containsItems() {
         boolean anyValid = false;
         for (ForgeDirection dir : ForgeDirection.VALID_DIRECTIONS) {
             // Avoid sending stuff into itself me network
-            if (checkValidSide(this.posInterface.getOffSet(dir).getTileEntity(), dir)
-                    && !isItemConduit(this.posInterface.getOffSet(dir).getTileEntity())) {
-                final int result = checkItemFluids(this.getSideFluid(dir), this.getSideItem(dir), dir.getOpposite());
-                if (result == 1) {
-                    return true;
-                }
-                if (result != 2) {
-                    anyValid = true;
-                }
+            TileEntity te = this.posInterface.getOffSet(dir).getTileEntity();
+            if (te == null || !checkValidSide(te, dir)
```

Comment:
Please make this logic a bit simplier or break it into multiple checks or make it more readable with parenthesis placement. Without extra parenthesis the second line is ambiguious to the reader.

Reply:
I changed logic, hope better.